tests/components/alarm_control_panel/test_device_action.py

```diff
@@ -266,42 +310,34 @@ def entity_reg(hass):
             ]
         },
     )
-    assert await async_setup_component(hass, DOMAIN, {DOMAIN: {CONF_PLATFORM: "test"}})
-    assert (
-        hass.states.get("alarm_control_panel.alarm_no_arm_code").state == STATE_UNKNOWN
-    )
+    assert await async_setup_component(hass, DOMAIN,
+                                       {DOMAIN: {
+                                           CONF_PLATFORM: "test"
+                                       }})
+    assert (hass.states.get("alarm_control_panel.alarm_no_arm_code").state ==
+            STATE_UNKNOWN)
 
     hass.bus.async_fire("test_event_arm_away")
     await hass.async_block_till_done()
-    assert (
-        hass.states.get("alarm_control_panel.alarm_no_arm_code").state
-        == STATE_ALARM_ARMED_AWAY
-    )
+    assert (hass.states.get("alarm_control_panel.alarm_no_arm_code").state ==
+            STATE_ALARM_ARMED_AWAY)
 
     hass.bus.async_fire("test_event_arm_home")
     await hass.async_block_till_done()
-    assert (
-        hass.states.get("alarm_control_panel.alarm_no_arm_code").state
-        == STATE_ALARM_ARMED_HOME
-    )
+    assert (hass.states.get("alarm_control_panel.alarm_no_arm_code").state ==
+            STATE_ALARM_ARMED_HOME)
 
     hass.bus.async_fire("test_event_arm_night")
     await hass.async_block_till_done()
-    assert (
-        hass.states.get("alarm_control_panel.alarm_no_arm_code").state
-        == STATE_ALARM_ARMED_NIGHT
-    )
+    assert (hass.states.get("alarm_control_panel.alarm_no_arm_code").state ==
+            STATE_ALARM_ARMED_NIGHT)
 
     hass.bus.async_fire("test_event_disarm")
     await hass.async_block_till_done()
-    assert (
-        hass.states.get("alarm_control_panel.alarm_no_arm_code").state
-        == STATE_ALARM_DISARMED
-    )
+    assert (hass.states.get("alarm_control_panel.alarm_no_arm_code").state ==
+            STATE_ALARM_DISARMED)
 
     hass.bus.async_fire("test_event_trigger")
     await hass.async_block_till_done()
-    assert (
-        hass.states.get("alarm_control_panel.alarm_no_arm_code").state
-        == STATE_ALARM_TRIGGERED
-    )
+    assert (hass.states.get("alarm_control_panel.alarm_no_arm_code").state ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4564454)